src/Traversal/Sdk/Sdk.targets

```diff
@@ -59,6 +63,10 @@
     <PackDependsOn>
       ResolveReferences;
     </PackDependsOn>
+
+    <PublishDependsOn>
+      Build;
```

Comment:
@jeffkl Could you please help review this? I'm not sure what should be put here, and just `Build` here seems works.

Reply:
I don't believe these are needed.  The VSTest target doesn't really run in the traversal projects.  Instead I think it should just run the VSTest target for each project which already will build the project before running the target.  Although technically this will run the Build target in the traversal project which can run extra before/after stuff.  So maybe we do need it?  Did you test the behavior out locally and it did what you want?

Reply:
I tried remove the Build target from VSTestDependsOn, but it seems failed to execute VSTest target on referenced projects. I roughly walk through the log but have no idea why it happens

Reply:
I guess it shouldn't hurt, since publish depends on build for each project anyway.  We'll just get it out there and fix it if people report problems.

Thanks!